assets/css/static-main.scss

```
@@ -247,4 +262,22 @@ a.ds-c-button.ds-u-margin-top--4:focus, button.accordion:focus, button.accordion
 
 .hex_background {
     background-image: url("/assets/img/hex_background.svg");
-}
\ No newline at end of file
+}
+
+table, th, td {
+    border: 1px solid black;
+    border-collapse: collapse;
+  }
+
+th, td {
+    padding: 15px;
+    text-align: left;
+}
+
+th {
+    background-color: lightgrey;
+}
+
+#cex {
```

```suggestion
.cex {
```
Change to a class to match HTML.